lang/rust/rust.py

```diff
@@ -0,0 +1,217 @@
+from talon import Module, Context, actions, ui, imgui, settings
+
+mod = Module()
+
+ctx = Context()
+ctx.matches = r'''
+mode: user.rust
+mode: user.auto_lang
+and code.language: rust
+'''
+
+ctx.lists['user.code_libraries'] = {
+    'eye oh': 'std::io',
+    'file system': 'std::fs',
+    'envy': 'std::env',
+    'collections': 'std::collections',
+}
+ctx.lists['user.code_functions'] = {
+    'drop': 'drop',
+    'print line': 'println!',
+    'error line': 'eprintln!',
+}
+
+ctx.lists['user.code_type'] = {
+    'you eight': 'u8',
+    'eye eight': 'i8',
+    'you sixteen': 'u16',
+    'eye sixteen': 'i16',
+    'you thirty two': 'u32',
+    'eye thirty two': 'i32',
+    'you sixty four': 'u64',
+    'eye sixty four': 'i64',
+    'you size': 'usize',
+    'eye size': 'isize',
+    'boolean': 'bool',
+    'string': 'str',
+    'owned strign': 'String',
+}
+
+@ctx.action_class('user')
+class UserActions:
+    def code_operator_lambda():
+        actions.insert('|| ')
```

Comment:
Maybe this should also do `actions.edit.left(); actions.edit.left()` to put the cursor between the vertical bars?